packages/mux-player/src/template.ts

```diff
@@ -108,23 +108,20 @@ export const content = (props: MuxTemplateProps) => html`
         : html``}
     </mux-video>
     ${isLiveOrDVR(props)
-      ? html`<button
```

Comment:
I think this is fine for a first pass, but I think ultimately, we'd want it to be in template itself replacing the slot https://github.com/muxinc/elements/blob/main/packages/mux-player/src/media-theme-mux.html

Reply:
Just had a chat about this with @cjpillsbury, I'm going to move it to the template before merging 🙂